pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -254,7 +254,7 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 func RangeSize(subnet *net.IPNet) int64 {
```

Comment:
Looking at the usage of this, the function panics (should be returning error, IMHO), if the subnet is too large, and if not, returns the number of hosts available, and the caller then checks if the subnet is too small. Maybe this should be changed to CheckSubnetSize() and return error if too large or too small, and caller can just check return (error) for non-nil?